packages/docusaurus-theme-search-algolia/package.json

```diff
@@ -12,6 +13,12 @@
     "directory": "packages/docusaurus-theme-search-algolia"
   },
   "license": "MIT",
+  "scripts": {
+    "build": "yarn build:server & yarn build:browser & yarn build:copy",
+    "build:server": "tsc --project tsconfig.server.json",
+    "build:browser": "tsc --project tsconfig.browser.json",
+    "build:copy": "node copyUntypedFiles.js"
+  },
```

Comment:
not sure if this is fine, there is no current example of package with build of ts/x files that support both server and browser code